server/apps/preferences.py

```diff
@@ -120,6 +120,13 @@ class PreferencesResource(Resource):
         'category': 'dateline'
     })
 
+    superdesk.register_default_user_preference('categories:preferred', {
```

Comment:
Sure, any concrete suggestions for the scenarios perhaps? If not, we'll just think and come up with something sensible tomorrow...